units/ships.json

```diff
@@ -0,0 +1,210 @@
+{
+    "data" : {
+        "Object_Type": "Vessel",
+        "Hud_Functionality": "1;1;1;1;1;1;1;1;1;1;1;1;1;1;1;1;1;1;1;1;1;1;1;1;1;1;1;1;1;1;1;1;1",
+        "Max_Hud_Functionality": "1;1;1;1;1;1;1;1;1;1;1;1;1;1;1;1;1;1;1;1;1;1;1;1;1;1;1;1;1;1;1;1;1",
+        "Lifesupport_Functionality" : "1",
+        "Max_Lifesupport_Functionality": "1",
+        "Comm_Functionality": "1",
+        "Max_Comm_Functionality": "1",
+        "FireControl_Functionality": "1",
+        "Max_FireControl_Functionality": "1",
+        "SPECDrive_Functionality": "1",
+        "Max_SPECDrive_Functionality" : "1",
+        "Tractorability": "pPiI",
+        "Jump_Drive_Delay": "1",
+        "Sound": ";;;;;;"
+    },
+    "units" : [
+        {
+            "data": {
+                "Key": "Areus",
+                "Directory": "./vessels/Areus",
+                "Name": "Areus",
+                "Object_Type": "Vessel",
+                "Combat_Role": "BOMBER",
+                "Textual_Description": "Aeran assault craft (fighter bomber)",
+                "STATUS": "MANEUVOUR",
+                "Hud_image": "areus-hud.spr",
+                "Unit_Scale": "1.25",
+                "Cockpit": "light_cockpit.cpt",
+                "CockpitZ": "16",
+                "Mesh": "{areus.bfxm;;}",
+                "Mass": "76",
+                "Fuel_Capacity": "18",
+                "Hull": "450",
+                "Outsystem_Jump_Cost": "200",
+                "Warp_Usage_Cost": "200",
+                "Afterburner_Type": "1",
+                "Afterburner_Usage_Cost": "4",
+                "Maneuver_Yaw": "24000",
+                "Maneuver_Pitch": "24000",
+                "Maneuver_Roll": "50000",
+                "Yaw_Governor": "62",
+                "Pitch_Governor": "70",
+                "Roll_Governor": "62",
+                "Afterburner_Accel": "21600",
+                "Forward_Accel": "15000",
+                "Retro_Accel": "10000",
+                "Left_Accel": "4000",
+                "Right_Accel": "4000",
+                "Top_Accel": "4000",
+                "Bottom_Accel": "4000",
+                "Afterburner_Speed_Governor": "220",
+                "Default_Speed_Governor": "220",
+                "Hold_Volume": "50",
+                "Prohibited_Upgrades": "upgrades/Overdrive:1;upgrades/Capacitors:1;upgrades/Reactors:1;upgrades/Armor:1;upgrades/Sensors:1;upgrades/Shield_Systems:1;upgrades/Passenger_Quarters:1;upgrades/Repair_Systems:1;upgrades/ECM_Systems:1",
```

Comment:
It would be good to break these down as well. guessing that needs another round of refactor so not blocking, just noting.